lib/mysql.js

```diff
@@ -351,7 +351,7 @@ MySQL.prototype.fromColumnValue = function(prop, val) {
         val = String(val);
         break;
       case 'Date':
-        val = new Date(val.toString().replace(/GMT.*$/, 'GMT'));
+        //val = new Date(val.toString().replace(/GMT.*$/, 'GMT'));
```

Comment:
If this line is no longer needed, then you can remove it instead of commenting it.